### PR TITLE
8 setup exception tracking

### PR DESCRIPTION
**Pull request by alishaevn**
# Story
configure exception tracking with sentry

# Expected Behavior Before Changes
- sentry was set up for react. turns out, there's a different way to set it up for next js

# Expected Behavior After Changes
- remove @sentry/react and @sentry/tracing
- set up sentry for next js
- intentionally left sentry setup comments so future internal/external devs know their options.

# Screenshots / Video

image

# resources
- https://docs.sentry.io/platforms/javascript/guides/nextjs/

### CODE REVIEW

File: ops/staging-deploy.tmpl.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ nextPublicToken: $NEXT_PUBLIC_TOKEN
 nextAuthSecret: $NEXTAUTH_SECRET
 clientSecret: $CLIENT_SECRET
 clientId: $CLIENT_ID
-sentryDsn: 'https://6c537d3cdd954401a893831313a976cc@o1008683.ingest.sentry.io/4504805305810944'
+sentryDsn: 'https://78ce09ae2f694f0e8981cf5cec6fc2c7@o1008683.ingest.sentry.io/4504810271408128'
```
Comment (alishaevn) [inline ops/staging-deploy.tmpl.yaml:10]: created a new sentry project, so there's a different dns.

File: .github/workflows/test-suite.yml [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-          # Defaults to the user or organization that owns the workflow file
-          scope: '@scientist-softserv'
-          node-version-file: package.json
-      - name: Authenticate with private GPR package
-        run: echo "//npm.pkg.github.com/:_authToken=${{ secrets.GITHUB_TOKEN }}" > $NPM_CONFIG_USERCONFIG
-      - run: yarn install
-      - run: yarn test
-      - run: yarn build
+# TODO(alishaevn): remove the references to the github package registry
+# TODO(alishaevn): run the cypress tests during this action
```
Comment (alishaevn) [inline .github/workflows/test-suite.yml:2]: ref: https://github.com/scientist-softserv/webstore/issues/275

File: .env [hunk 3]
```diff
@@ -5,4 +5,4 @@ NEXT_PUBLIC_PROVIDER_NAME=beachsidebiotech
 NEXT_PUBLIC_PROVIDER_ID=5159
 NEXT_PUBLIC_SCIENTIST_API_VERSION=v2
 
-SENTRY_DSN=https://6c537d3cdd954401a893831313a976cc@o1008683.ingest.sentry.io/4504805305810944
+SENTRY_DSN=https://78ce09ae2f694f0e8981cf5cec6fc2c7@o1008683.ingest.sentry.io/4504810271408128
```
Comment (alishaevn) [inline .env:8]: created a new sentry project, so there's a different dns.

**Review by aprilrieger — APPROVED** [review]
Great work as always Alisha!

**Pull request merged**